packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
+  active,
+  title,
+  subtitle,
+  subitem,
+  miniitem,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
+  onClick,
+}) => (
+  <div
+    onClick={onClick}
+    className={cx({
+      ListItem: true,
+      "ListItem--subitem": subitem && !miniitem,
+      "ListItem--miniitem": miniitem,
+      "is-active": active,
+      "is-clickable": !!onClick,
+      "ListItem-divider": !hideDivider,
+    })}
+  >
+    {leftIcon && (
+      <div className="ListItem-leftIcon">
+        <Icon icon={leftIcon} />
+      </div>
+    )}
+    <div>
+      <div className="ListItem-title">{title}</div>
+      {!!subtitle && <div className="ListItem-subtitle">{subtitle}</div>}
```

Comment:
```suggestion
      {subtitle && <p className="ListItem-subtitle">{subtitle}</p>}
```

Reply:
`!!` ✅ 
`<p>` messes with my styles :/

Reply:
CSS to the rescue 😉

Reply:
Yeah :) But I'm rather like - why `<p>` instead of `<div>`? `<div>` works just as well and I don't have to override `<p>`'s styles :)

Reply:
Semantic my friend, semantic 👌